setupbase.py

```diff
@@ -253,24 +250,25 @@ def run(self):
 class UpdateCodiconCommand(setuptools.Command):
     """A custom command to make updating Microsoft's Codicons easy!"""
     description = 'Try to update the Codicon font data in the project.'
-    user_options = []
+    user_options = [('msc-version=', None, 'Codicon version.')]
 
     CHARMAP_PATH = os.path.join(
         HERE, "qtawesome", "fonts", "codicon-charmap-{version}.json"
     )
     TTF_PATH = os.path.join(
         HERE, "qtawesome", "fonts", "codicon-{version}.ttf"
     )
-    DOWNLOAD_URL_TTF = 'https://raw.githubusercontent.com/microsoft/vscode-codicons/main/dist/codicon.ttf'
-    DOWNLOAD_URL_CSV = 'https://raw.githubusercontent.com/microsoft/vscode-codicons/main/dist/codicon.csv'
-    # At the time of writing this comment, vscode-codicons repo does not use git tags, but you can get the version from package.json:
-    DOWNLOAD_URL_JSON = 'https://raw.githubusercontent.com/microsoft/vscode-codicons/main/package.json'
+    DOWNLOAD_URL_TTF = 'https://raw.githubusercontent.com/microsoft/vscode-codicons/{version}/dist/codicon.ttf'
+    DOWNLOAD_URL_CSS = 'https://raw.githubusercontent.com/microsoft/vscode-codicons/{version}/dist/codicon.css'
+    DOWNLOAD_URL_JSON = 'https://raw.githubusercontent.com/microsoft/vscode-codicons/{version}/package.json'
 
     def initialize_options(self):
-        """Required by setuptools."""
+        """Set default values for the command options."""
+        self.msc_version = ''
 
     def finalize_options(self):
-        """Required by setuptools."""
+        """Validate the command options."""
+        assert bool(self.msc_version), 'Codicons version is mandatory for this command.'
```

Comment:
```suggestion
        assert bool(
            self.msc_version
        ), 'Codicons version is mandatory for this command.'
```

Make this fit in 79 columns